saskatoon/harvest/api.py

```diff
@@ -221,3 +224,188 @@ def list(self, request, *args, **kwargs):
                                  'title': _("New Person")
                                  }
                          })
+
+
+class StatsViewset(LoginRequiredMixin, viewsets.ModelViewSet):
```

Comment:
Also the choice of using a ModelViewSet here might be questionable since you are not actually displaying a list of (filtered) harvests and you seem to only need a single view (with a kwarg argument). ViewSets are usually used to implement all CRUD operations. I feel like a GenericAPIView would be enough in your case.